Examples/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
+*/
+!.gitignore
```

Comment:
See my other comment in other PR that you would typically add these to the `.gitignore` in the project root.

Reply:
Yes! I actually decided to add a README and keep the examples in, it seems like that would be helpful for people to have if they're getting to know the program.

Reply:
Definitely in favor of keeping the examples in for users! 👍